bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
nick - the query printed is *exactly* what is passed into the code that turns it into an xmlrpc request.

I've repeatedly said it is fine to *also* print out the right query but please don't keep suggesting making the debug output wrong. thank you.

Also, if you could provide info where you found the info on these fields since I would really like to know why its treated so differently.